sdk/src/main/scala/com/stratio/sparkta/sdk/OperatorConstants.scala

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.sdk
+
+object OperatorConstants {
+
+  //FIXME: Do we really need this?
+  final val EmptyString = ""
+  final val SpaceSeparator = " "
+  final val One = 1
+  final val Zero = 0
+  final val UnderscoreSeparator = "_"
+  final val NullValue = "None"
```

Comment:
One and NullValue constants are used just in one class. Please move this constants there